client/src/redux/project/project.types.ts

```diff
@@ -1,15 +1,21 @@
 export interface Project {
-	_id: string;
+	_id?: string;
 	title: string;
 	description: string;
-	image: string;
+	image?: string;
 }
 
 export const SET_ALL_PROJECTS = 'SET_ALL_PROJECTS';
-export const CREATE_PROJECT = 'CREATE_PROJECT';
+export const CREATE_PROJECT_START = 'CREATE_PROJECT_START';
+export const CREATE_PROJECT_SUCCESS = 'CREATE_PROJECT_SUCCESS';
```

Comment:
I decided to  rename them like this because the _START action gets caught by a saga rather than a reducer because  it is asynchronous (requires fetching from the server)